src/main/java/frc/robot/RobotContainer.java

```diff
@@ -33,16 +34,23 @@ public class RobotContainer {
   private final Limelight limelight;
 
   private final GripperSystem gripperSystem;
-  private final XboxController driver = new XboxController(OperatorConstants.kDriverControllerPort);
-  private final JoystickButton xButton = new JoystickButton(driver, XboxController.Button.kX.value);
+  private final XboxController operator;
+  private final JoystickButton xButton;
+  private final Joystick driverLeft;
+  private final Joystick driverRight;
 
   // hardware connection check stuff
   private final NetworkTable hardware = NetworkTableInstance.getDefault().getTable("Hardware");
 
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
+  operator = new XboxController(OperatorConstants.OP_CONTROLLER);
+  xButton = new JoystickButton(operator, XboxController.Button.kX.value);
+  driverLeft = new Joystick(OperatorConstants.DRIVER_LEFT_PORT);
+  driverRight = new Joystick(OperatorConstants.DRIVER_RIGHT_PORT);
+
```

Comment:
fix indentation

Reply:
This pull request needs to be done before others to prevent merge conflicts, start a new issue/pull request to fix this instead